code/modules/reagents/Chemistry-Goon-Medicine.dm

```diff
@@ -378,4 +378,218 @@ datum/reagent/morphine/on_mob_life(var/mob/living/M as mob)
+
+datum/reagent/atropine
+	name = "Atropine"
+	id = "atropine"
+	description = "1 TOX damage if used over -60 health. Causes dizziness and confusion. If under -25 health, heals 3 BRUTE + 3 BURN. Attempts to cap OXY damage at 65 and LOSEBREATH at 5."
+	reagent_state = LIQUID
+	color = "#C8A5DC" // rgb: 200, 165, 220
+	metabolization_rate = 0.2
+
+datum/reagent/atropine/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	if(M.health > -60)
+		M.adjustToxLoss(1*REM)
+	if(M.health < -25)
+		M.adjustBruteLoss(3*REM)
+		M.adjustFireLoss(3*REM)
+	if(M.oxyloss > 65)
+		M.setOxyLoss(65)
+	if(M.losebreath > 5)
+		M.losebreath = 5
+	if(prob(30))
+		M.Dizzy(5)
+		M.Jitter(5)
+	..()
+	return
+
+/datum/chemical_reaction/atropine
+	name = "Atropine"
+	id = "atropine"
+	result = "atropine"
+	required_reagents = list("ethanol" = 1, "acetone" = 1, "diethylamine" = 1, "phenol" = 1, "sacid" = 1)
+	result_amount = 5
+
+datum/reagent/epinephrine
+	name = "Epinephrine"
+	id = "epinephrine"
+	description = "Reduces most of the knockout/stun effects, minor stamina regeneration buff. Attempts to cap OXY damage at 35 and LOSEBREATH at 3. If health is between -10 to -65, heals 1 TOX, 1 BRUTE, 1 BURN."
+	reagent_state = LIQUID
+	color = "#C8A5DC" // rgb: 200, 165, 220
+	metabolization_rate = 0.2
+
+datum/reagent/epinephrine/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	if(M.health > -10 && M.health < -65)
+		M.adjustToxLoss(-1*REM)
+		M.adjustBruteLoss(-1*REM)
+		M.adjustFireLoss(-1*REM)
+	if(M.oxyloss > 35)
+		M.setOxyLoss(35)
+	if(M.losebreath > 3)
+		M.losebreath = 3
+	M.adjustStaminaLoss(-1*REM)
+	if(prob(30))
+		M.AdjustParalysis(-1)
+		M.AdjustStunned(-1)
+		M.AdjustWeakened(-1)
+		M.sleeping -= 1
+	..()
+	return
+
+/datum/chemical_reaction/epinephrine
+	name = "Epinephrine"
+	id = "epinephrine"
+	result = "epinephrine"
+	required_reagents = list("phenol" = 1, "acetone" = 1, "diethylamine" = 1, "oxygen" = 1, "chlorine" = 1, "hydrogen" = 1)
+	result_amount = 6
+
+datum/reagent/strange_reagent
+	name = "Strange Reagent"
+	id = "strange_reagent"
+	description = "A miracle medical chem, this little beauty can bring the dead back to life! ...Well, if you're careful that is. If the cadaver is rotten or has too much BRUTE/BURN damage, SR will gib them on use."
+	reagent_state = LIQUID
+	color = "#C8A5DC" // rgb: 200, 165, 220
+	metabolization_rate = 0.2
+
+datum/reagent/strange_reagent/reaction_mob(var/mob/living/M as mob, var/method=TOUCH, var/volume)
+	if(M.bruteloss > 80 || M.fireloss > 80)
+		if(ishuman(M) || ismonkey(M))
+			var/mob/living/carbon/C_target = M
+			var/obj/item/organ/brain/B = C_target.getorgan(/obj/item/organ/brain)
+			if(B)
+				B.loc = get_turf(C_target)
+				B.transfer_identity(C_target)
+				C_target.internal_organs -= B
+			M.gib()
+	else if(M.stat == DEAD)
+		M.stat = CONSCIOUS
+		M.adjustOxyLoss(-50*REM)
+		M.adjustBruteLoss(-50*REM)
+		M.adjustToxLoss(-50*REM)
+		M.adjustFireLoss(-50*REM)
```

Comment:
Strange reagent basically works like this: you have to heal ALL BURN and BRUTE damage before using it to revive the person, meaning you're going to have to slap a ton of silver sulfa or styptic on the person while they're dead to heal the damage.

If the person has left the game or they still have damage left, then strange reagent will out and out gib the person you're using it on. It's not an easy chem to use by any stretch of the imagination and hardly ever sees use on Goon because of it.